.gitignore

```diff
@@ -8,3 +8,4 @@ userland/bin/
 *.so
 *.tar
 .gdb_history
+.DS_Store
```

Comment:
This isn't needed in a local gitignore file. It should be added to your [global gitignore file](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) instead.

Reply:
I'll revert this change.